server/src/graphql/movie/mutations/CreateMovie.ts

```diff
@@ -13,11 +14,20 @@ export default mutationWithClientMutationId({
       type: new GraphQLNonNull(GraphQLString),
     },
   },
-  mutateAndGetPayload: async ({ title, genre }) => {
+  mutateAndGetPayload: async ({ title, genre }, ctx) => {
+    const header = ctx.request.header.authorization;
+
+    if (!header) {
+      return { error: "You are not logged in. Please, sign in" };
+    }
+    const token = header.split(" ");
+    const verifyJWT = verify(token[1], process.env.JWT_SECRET);
+    if (!verifyJWT) {
+      return { error: "You are not logged in. Please, sign in" };
+    }
```

Comment:
- move this check to the getContext
- then, there get all those data and from it get the user
- set the user on the context
- if do not exist the user it is not logged

Reply:
done